ibis/backends/dask/execution/util.py

```diff
@@ -218,29 +223,65 @@ def compute_sort_key(
     name = ibis.util.guid()
     if key.name in data:
         return name, data[key.name]
+    if isinstance(key, str):
+        return key, None
+    elif key.name in data:
```

Comment:
Isn't this already handled in line 224? https://github.com/ibis-project/ibis/pull/5747/files#diff-7560a298b0242d66bc062059c8a1189235eea9088c8174ce22b7154caf1d629fR224

Reply:
I'm not sure how we would ever hit this.

Reply:
Yeah, i think this got introduced from resolving some merge conflicts poorly.